cnkzsjcnaksjcnaskjnckasjns

KeaganQuadras opened the pull request:
sxas

KaneDenzil commented:
@KeaganQuadras  Add proper Heading for the PR and the description. Not sure what have you built.

KeaganQuadras commented:
- Added a new header 
- Edited the description for product management 
- changed the font by using <u>  and ,<b> tags for both the modules headings
- added / removed some points in both the modules
- added new headings [how will you benefit, pedagogy, program delivery] and the given points 
- edited class schedule 
- added new fields [eligibility, admission criteria]

KeaganQuadras commented:
- Added a new header 
- Edited the description for product management 
- changed the font by using <u>  and ,<b> tags for both the modules headings
- added / removed some points in both the modules
- added new headings [how will you benefit, pedagogy, program delivery] and the given points 
- edited class schedule 
- added new fields [eligibility, admission criteria]
